includes/forms.inc

```diff
@@ -54,13 +54,13 @@ function islandora_ip_embargo_settings_form($form, &$form_state) {
   $form['islandora_ip_embargo_overlay_text_color'] = array(
     '#type' => 'textfield',
     '#title' => t('Embargoed item overlay text color'),
-    '#description' => t('Overlay text color in hexideciamal format (E.g. #FF0000).'),
-    '#maxlength' => 7,
+    '#description' => t('Overlay text color in hexideciamal format (E.g. FF0000).'),
```

Comment:
Since the # was in there before, we should keep it in there. Don't wanna break existing peoples' things. We should instead just trim # off when validating